### DISCUSSION TITLE
Customizable statusbar

### DISCUSSION THREAD

**Discussion by davidolrik** [mcndt/obsidian-toggl-integration#2] [Feature Requests]
It would be nice to show the Project name in the status bar instead of the description.

If we could use a template in the status bar, we could change it to fit each user of the plugin.

Some ideas about what to customize

- Show Project name
- Show description
- Show time in hours and minutes
- Change the prefix from "Timer" to ⏱ or ⏳

**Comment by davidolrik** [comment]
It would also be cool if I could show the Project in the sidebar, as I don't use descriptions much.
Right now it just says `undefined`.

**Reply by mcndt** [reply]
Hey David, thanks for letting me know how I can improve the plugin to suit your workflow! I like your idea for templating the toolbar message, I think in the Obsidian community especially this kind of customizability is seen a lot. 

For now I'll make a patch such that descriptionless time entries don't say "undefined" (never a good look), and see if I can add a quick setting for choosing whether you wish to show project or description.

**Reply by mcndt** [reply]
Patch is now live! You can pull it inside Obsidian through the update plugins button in the settings.

For now I just showed "no description" instead of the undefined, to match the toolbar. In a future update I'll focus on making everything a little more customizable.
